lang/cs/Org.Apache.REEF.Tests/Functional/Bridge/TestCloseTask.cs

```diff
@@ -0,0 +1,171 @@
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using Org.Apache.REEF.Common.Context;
+using Org.Apache.REEF.Common.Tasks;
+using Org.Apache.REEF.Driver;
+using Org.Apache.REEF.Driver.Context;
+using Org.Apache.REEF.Driver.Evaluator;
+using Org.Apache.REEF.Driver.Task;
+using Org.Apache.REEF.Tang.Annotations;
+using Org.Apache.REEF.Tang.Implementations.Tang;
+using Org.Apache.REEF.Tang.Interface;
+using Org.Apache.REEF.Tang.Util;
+using Org.Apache.REEF.Utilities.Logging;
+using Xunit;
+
+namespace Org.Apache.REEF.Tests.Functional.Bridge
+{
+    /// <summary>
+    /// This test is to close a running task from driver
+    /// </summary>
+    public sealed class TestCloseTask : ReefFunctionalTest
+    {
+        private static readonly Logger Logger = Logger.GetLogger(typeof(TestCloseTask));
+
+        private const string DisposeMessageFromDriver = "DisposeMessageFromDriver";
+        private const string CompletedValidationMessage = "CompletedValidationmessage";
+
+        public TestCloseTask()
+        {
+            Init();
+        }
+
+        /// <summary>
+        /// This test is to close a running task over the bridge
+        /// </summary>
+        [Fact]
+        public void TestStopTaskOnLocalRuntime()
+        {
+            string testFolder = DefaultRuntimeFolder + Guid.NewGuid().ToString("N").Substring(0, 4);
+            TestRun(DriverConfigurations(), typeof(StopTaskHandlers), 1, "testStopTask", "local", testFolder);
+            ValidateSuccessForLocalRuntime(1, testFolder: testFolder);
+            ValidateMessageSuccessfullyLoggedForDriver(CompletedValidationMessage, testFolder, 1);
+            var messages = new List<string>();
+            messages.Add(DisposeMessageFromDriver);
+            ValidateMessageSuccessfullyLogged(messages, "Node-*", EvaluatorStdout, testFolder, 1);
+            CleanUp(testFolder);
+        }
+
+        /// <summary>
+        /// Driver configuration for the test driver
+        /// </summary>
+        /// <returns></returns>
+        public IConfiguration DriverConfigurations()
+        {
+            var helloDriverConfiguration = DriverConfiguration.ConfigurationModule
+                .Set(DriverConfiguration.OnDriverStarted, GenericType<StopTaskHandlers>.Class)
+                .Set(DriverConfiguration.OnEvaluatorAllocated, GenericType<StopTaskHandlers>.Class)
+                .Set(DriverConfiguration.OnContextActive, GenericType<StopTaskHandlers>.Class)
+                .Set(DriverConfiguration.OnTaskRunning, GenericType<StopTaskHandlers>.Class)
+                .Set(DriverConfiguration.OnTaskCompleted, GenericType<StopTaskHandlers>.Class)
+                .Build();
+
+            return TangFactory.GetTang().NewConfigurationBuilder(helloDriverConfiguration).Build();
```

Comment:
why the need for the builder here? Can't you just `return helloDriverConfiguration `?